wrappers/node/test/suite1/ariesvcx-connection.test.ts

```diff
@@ -1,13 +1,23 @@
 import '../module-resolver-helper'
 
+// @ts-ignore
 import { assert } from 'chai'
+// @ts-ignore
 import { connectionCreate, connectionCreateConnect, dataConnectionCreate } from 'helpers/entities'
-import { INVITE_ACCEPTED_MESSAGE, INVITE_REDIRECTED_MESSAGE, INVITE_DETAILS } from 'helpers/test-constants'
+// @ts-ignore
+import {// @ts-ignore
+  INVITE_ACCEPTED_MESSAGE, // @ts-ignore
+  INVITE_DETAILS,// @ts-ignore
+  INVITE_REDIRECTED_MESSAGE,
+  PROTOCOL_TYPE_ARIES
+} from 'helpers/test-constants'
```

Comment:
Maybe we should type the constants as well?

Reply:
done